subworkflows/local/fimo.nf

```diff
@@ -14,25 +13,18 @@ workflow FIMO {
         fasta
         tf_ranking
         enhancer_regions
-        pwm
+        motifs_meme
 
     main:
         ch_versions = Channel.empty()
 
-        JASPAR_MAPPING(tf_ranking, pwm)
-
-        JASPAR_DOWNLOAD()
-
-        FILTER_MOTIFS(JASPAR_MAPPING.out.jaspar_ids, JASPAR_DOWNLOAD.out.motifs)
+        JASPAR_MAPPING(tf_ranking, motifs_meme)
+        FILTER_MOTIFS(JASPAR_MAPPING.out.jaspar_ids, motifs_meme)
```

Comment:
`JASPAR_MAPPING` and `FILTER_MOTIFS` aim to keep only the motifs for transcription factors found significant by the pipeline. Splitting this into two processes made sense in the original implementation with a dedicated download from JASPAR, but now this should be done in a single process.